src/emmc.c

```diff
@@ -531,6 +550,45 @@ pu_emmc_parse_emmc_bootpart(PuEmmc *emmc,
     return TRUE;
 }
 
+static gboolean
+pu_emmc_parse_clean(PuEmmc *emmc,
+                    GHashTable *root,
+                    GError **error)
+{
+    PuConfigValue *value_clean = g_hash_table_lookup(root,"clean");
+    GList *clean;
+
+    g_return_val_if_fail(emmc != NULL, FALSE);
+    g_return_val_if_fail(root != NULL, FALSE);
+    g_return_val_if_fail(error == NULL || *error == NULL, FALSE);
+
+    if (!value_clean) {
+        g_debug("No entry 'clean' found. Skipping...");
+        return TRUE;
+    }
+
+    if (value_clean->type != PU_CONFIG_VALUE_TYPE_SEQUENCE) {
+        g_set_error(error, PU_ERROR, PU_ERROR_EMMC_PARSE,
+                    "'clean' is not a sequence");
+        return FALSE;
+    }
+    clean = value_clean->data.sequence;
+
+    for (GList *b = clean; b != NULL; b = b->next) {
```

Comment:
You may want to use `c` instead `b` for the GList pointer name ("b" for raw binary, "c" for cleaning spot).